src/sentry/static/sentry/less/includes/bootstrap/type.less

```diff
@@ -7,10 +7,6 @@
 // Body text
 // -------------------------
 
-p {
-  margin: 0 0 (@line-height-computed / 2);
-}
```

Comment:
These styles are being overridden in base styles, so bye. Also the rest of this file needs to go, but again that’s for later.